lib/mayaUsd/ufe/UsdTransform3d.cpp

```diff
@@ -92,107 +92,107 @@ void UsdTransform3d::setItem(const UsdSceneItem::Ptr& item)
 
 const Ufe::Path& UsdTransform3d::path() const
 {
-	return fItem->path();
+    return fItem->path();
 }
 
 Ufe::SceneItem::Ptr UsdTransform3d::sceneItem() const
 {
-	return fItem;
+    return fItem;
 }
 
 #if UFE_PREVIEW_VERSION_NUM >= 2013
 Ufe::TranslateUndoableCommand::Ptr UsdTransform3d::translateCmd(double x, double y, double z)
 {
-	return UsdTranslateUndoableCommand::create(fItem, x, y, z);
+    return UsdTranslateUndoableCommand::create(path(), x, y, z);
 }
 #endif
 
 void UsdTransform3d::translate(double x, double y, double z)
 {
-	translateOp(fPrim, fItem->path(), x, y, z);
+    translateOp(fItem->prim(), fItem->path(), x, y, z);
 }
 
 Ufe::Vector3d UsdTransform3d::translation() const
 {
-	double x{0}, y{0}, z{0};
-	const TfToken xlate("xformOp:translate");
-	if (fPrim.HasAttribute(xlate))
-	{
-		// Initially, attribute can be created, but have no value.
-		GfVec3d v;
-		if (fPrim.GetAttribute(xlate).Get<GfVec3d>(&v,getTime(path())))
-		{
-			x = v[0]; y = v[1]; z = v[2];
-		}
-	}
-	return Ufe::Vector3d(x, y, z);
+    double x{0}, y{0}, z{0};
+    const TfToken xlate("xformOp:translate");
+    if (fItem->prim().HasAttribute(xlate))
+    {
+        // Initially, attribute can be created, but have no value.
+        GfVec3d v;
+        if (fItem->prim().GetAttribute(xlate).Get<GfVec3d>(&v,getTime(path())))
+        {
+            x = v[0]; y = v[1]; z = v[2];
+        }
+    }
+    return Ufe::Vector3d(x, y, z);
 }
 
 #if UFE_PREVIEW_VERSION_NUM >= 2013
 Ufe::Vector3d UsdTransform3d::rotation() const
 {
-	double x{0}, y{0}, z{0};
-	const TfToken rotXYZ("xformOp:rotateXYZ");
-	if (fPrim.HasAttribute(rotXYZ))
-	{
-		// Initially, attribute can be created, but have no value.
-		GfVec3f v;
-		if (fPrim.GetAttribute(rotXYZ).Get<GfVec3f>(&v,getTime(path())))
-		{
-			x = v[0]; y = v[1]; z = v[2];
-		}
-	}
-	return Ufe::Vector3d(x, y, z);
+    double x{0}, y{0}, z{0};
+    const TfToken rotXYZ("xformOp:rotateXYZ");
+    if (fItem->prim().HasAttribute(rotXYZ))
```

Comment:
Create inline prim() accessor.

Reply:
addressed in: 78d393fcecadb769847a5ba66ef00286969381d7